src/parking/controls/editor/editor-form.ts

```diff
@@ -329,6 +385,11 @@ const conditionValues = [
     'residents',
     'customers',
     'private',
+    'disabled',
+    'no_parking',
+    'no_standing',
+    'no_stopping',
+    'no',
 ]
```

Comment:
Should `free; residents` be added?

In Australia, a common scenario is:
- Free parking
- Time limited for public to a few hours
- Unlimited time for residents (permit holders).

I believe this is covered by the tagging guidelines like so:

```
[parking:condition:right](https://wiki.openstreetmap.org/wiki/Key:parking:condition:right)=ticket; residents (This is a paid parking with no other restrictions. Residents may also park here with a permit.)
[parking:condition:right:conditional](https://wiki.openstreetmap.org/w/index.php?title=Key:parking:condition:right:conditional&action=edit&redlink=1)=no_parking @ (Oct 1-Apr 30: Mo 10:00-15:00) (You may not park here on Mondays 10-15 during October-April.)
[parking:condition:right:residents](https://wiki.openstreetmap.org/w/index.php?title=Key:parking:condition:right:residents&action=edit&redlink=1)=Ku (Residents need permit "Ta".)
```

(Taken from second example on https://wiki.openstreetmap.org/wiki/Key:parking:condition/complex)

Reply:
It too future task